README.md

```diff
@@ -0,0 +1,197 @@
+
+Your staging location will be:
+
+```bash
+gs://your-bucket-name/the-folder-that-you-might-have-created
+```
+
+![setting staging location][bucket1]
+
+![setting staging location][bucket2]
+
+#### Bigtable
+
+After enabling Bigtable, we'll need to create an instance where our table will be stored. And then create the actual table.
+
+##### Creating an instance
+
+Go to https://console.cloud.google.com/bigtable/instances?project=YOUR-PROJECT-ID , again taking care to change it with your project id. Click "Create instance" and fill the appropriate details. The name is, again, important. We used "test-instance" in our example. We suggest you to select HDD as a storage type as it is much cheaper, and this is for test-purposes only.
+                  
+|                                            |                                            |
+|:------------------------------------------:|:------------------------------------------:|
+| ![creating a bigtable instance][bigtable1] | ![creating a bigtable instance][bigtable2] |
+
+##### Creating a table
+
+We usually do this through an HBase shell inside Google Cloud Shell ([reference](https://cloud.google.com/bigtable/docs/quickstart-hbase)).  
+
+Go to your Google Cloud Dashboard (https://console.cloud.google.com/home/dashboard?project=YOUR-PROJECT-ID) and click on the little terminal symbol on the top right corner. That will boot your Google Shell. Wait a bit and then run the following commands:
+
+```bash
+$  curl -f -O https://storage.googleapis.com/cloud-bigtable/quickstart/GoogleCloudBigtable-Quickstart-0.9.5.1.zip
+$ unzip GoogleCloudBigtable-Quickstart-0.9.5.1.zip
+$ cd quickstart
+$ ./quickstart.sh
+```
+
+This will boot the HBase shell. You'll then issue a ```create``` command, followed by 2 quoted arguments, separated by an argument. These are the table name, and the column family. In our case, we used "test-table" as the table name and "cf1" as the column family:
+
+```
+hbase-shell> create "test-table", "cf1"
+```
+
+To see the data written in the table, later, you'll run the following command (again, inside the HBase shell):
+
+```
+hbase-shell> scan "test-table"
+```
+
+### 2. Sending events to Pub/Sub
+
+At this step, you've already created a Cloud Pub/Sub topic, either via the web interface or the helper script we provide in this repo. In our case, the topic is called "test-topic".
+
+To start the event generator, simply run the following command, on the repo's root:
+
+```bash
+$ inv generate_events --topic-name=test-topic
+```
+
+This will run an infinite loop, continuously publishing randomly generated events to our Pub/Sub topic. To stop it, Ctrl+C .
+
+### 3. Running our job on Dataflow
+
+You can either download a 'fat-jar' assembled by us, or build it yourself (if you have Scala and SBT installed).
```

Comment:
Also you mention that a fat jar is available for download but there are no mentions of where to download it from